core-v1/src/main/scala/algebra/Expr.scala

```diff
@@ -326,6 +341,111 @@ object Expr {
       opW: OP[W[Window[V]]],
       opB: OP[W[Boolean]],
     ): I ~:> W[Boolean]
+
+    def visitZipToShortestHList[I, W[+_], WL <: HList, UL <: HList](
+      expr: Expr.ZipToShortestHList[I, W, WL, UL, OP],
+    )(implicit
+      zip: ZipToShortest.Aux[W, WL, OP, UL],
+      opO: OP[W[UL]],
+    ): I ~:> W[UL]
+  }
+
+  abstract class ProxyVisitor[G[-_, +_], H[-_, +_], OP[_]](protected val underlying: Visitor[G, OP])
+    extends Visitor[H, OP] {
```

Comment:
This should be a `class` and not an `abstract class` to avoid issues with missing proxy methods.